Dockerfile

```diff
@@ -14,7 +14,7 @@ ENV PATH=/root/.cargo/bin:$PATH
 
 # Install datadog_lambda and dependencies from local
 COPY . .
-RUN pip install . ddtrace==2.21.3 -t ./python/lib/$runtime/site-packages
+RUN pip install --no-cache-dir . -t ./python/lib/$runtime/site-packages
```

Comment:
Is this safe to do? What's the implication of this? Do you remember why we were pinning before?

There was a pretty good reason about this from the past

Reply:
for `ddtrace==2.21.3` it's in the description.
 For `--no-cache-dir`... i actually wonder how it was added ... my local change didn't have it

Reply:
Ah, that's the reason, you were pinning in the past you wanted to make sure the fix was included for dynamodb, sg